online/src/main/scala/ai/chronon/online/LRUCache.scala

```diff
@@ -0,0 +1,57 @@
+package ai.chronon.online
+
+import com.github.benmanes.caffeine.cache.{Caffeine, Cache => CaffeineCache}
+
+/**
+  * Utility to create a cache with LRU semantics.
+  *
+  * The original purpose of having an LRU cache in Chronon is to cache KVStore calls and decoded IRs
+  * in the Fetcher. This helps decrease to feature serving latency.
+  */
+object LRUCache {
```

Comment:
Caffeine is technically not LRU but it's similar. I think naming this `LRUCache` makes it easy to understand what it does (and to contrast it with the existing `TTLCache`).